notebooks/7-rotation-speeds/interm_rotation_rate_solutions.ipynb

```diff
@@ -0,0 +1,3462 @@
+{
```

Comment:
<p>If people did this exercise in NB order, they might be confused: it's "not as tight a relationship" as....? As the plot we generate at the last step of the main Notebook? If so, let's just say that the relationship doesn't look to be a good fit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>